aspnetcore/fundamentals/metapackage-app.md

```diff
@@ -0,0 +1,58 @@
+
+The [Microsoft.AspNetCore.App](https://www.nuget.org/packages/Microsoft.AspNetCore.App) metapackage for ASP.NET Core includes:
+
+* All supported packages by the ASP.NET Core team.
+* All supported packages by the Entity Framework Core. 
+* Internal dependencies used by ASP.NET Core and Entity Framework Core.
+* 3rd-party dependencies used by ASP.NET Core and Entity Framework Core deemed necessary to ensure the major frameworks features function.
+
+All the features of ASP.NET Core 2.1 and later and Entity Framework Core 2.1 and later are included in the `Microsoft.AspNetCore.App` package. The default project templates targeting ASP.NET Core 2.1 and later use this package. We recommend applications targeting ASP.NET Core 2.1+ and Entity Framework Core 2.1+ use the `Microsoft.AspNetCore.App` package.
+
+The version number of the `Microsoft.AspNetCore.App` metapackage represents the ASP.NET Core version and Entity Framework Core version (aligned with the .NET Core version).
+
+Using the the `Microsoft.AspNetCore.App` metapackage provides version restrictions that protect your app:
+
+* Other packages added to your app cannot change the version of packages included in `Microsoft.AspNetCore.App`.
+* Version consistency ensures a reliable experience. `Microsoft.AspNetCore.App` was designed to prevent untested version combinations of related bits being used together in the same app.
+
+Applications that use the `Microsoft.AspNetCore.App` metapackage automatically take advantage of the .NET Core shared framework. When you use the `Microsoft.AspNetCore.App` metapackage, **no** assets from the referenced ASP.NET Core NuGet packages are deployed with the application &mdash; the .NET Core shared framework contains these assets. The assets in the shared framework are precompiled to improve application startup time. For more information, see "shared framework" in [.NET Core distribution packaging](/dotnet/core/build/distribution-packaging).
+
+Packages in the .NET Core shared framework are excluded in published application output.
+
+The following *.csproj* file references the `Microsoft.AspNetCore.App` metapackage for ASP.NET Core:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+
+</Project>
+
+```
+
+The preceding markup doesn't specify a version number for the `Microsoft.AspNetCore.App` NuGet package. The ASP.NET Core 2.1+ templates generate this markup without a version number. Not specifying a version is convent early in development when you want the latest version. Most teams will add a version number when deploying the app to ensure that development, test, and production are all using the same version. With a fixed version number, updates to the host will not cause the app to use a newer version of the NuGet package.
```

Comment:
convent? Word choice is a little confusing. Also, even while deploying an app, we don't recommend users to add a version number. We want the version number to remain implicit in all scenarios. The shared framework roll forward mechanism will always run the app on the latest version of the installed shared framework. To guarantee the same version is used in dev, test, and production, you'll need to ensure the same version of the shared framework is installed in all environments.